helpers/Singular_Value_Calculator/svd.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+import pandas as pd
+import matplotlib as plt
+import sympy as smp
+from sympy.solvers import solve
+from sympy import Symbol
+
+def SVD_Setup(Matrix):
+    #Build array, and transpose of that array
+    Matrix=np.array(Matrix).copy()
+
+    Transpose=np.transpose(Matrix)
+    #Perform Matrix Multiplication to find AAt and AtA Matrices
+    Multiplied_U=np.dot(Matrix, Transpose)
+
+    Multiplied_V=np.dot(Transpose, Matrix)
+    #Find Eigenvalues and Eigenvectors of AAt and AtA
+    values_U,vectors_U=np.linalg.eig(Multiplied_U)
+
+    values_V,vectors_V=np.linalg.eig(Multiplied_V)
+    #Compute Singular Values, the square roots of the eignvalues of AtA.
+    Sing_Values=np.sqrt(values_V)
+
+    return Sing_Values
+
+    def Create_SV_Array(Sing_Values, Matrix):
```

Comment:
mind your tabbing. currently this function is defined inside the scope of SVD_Setup, and will not be accessable from outside this file.